db/model/Gdoc/Gdoc.ts

```diff
@@ -709,6 +709,21 @@ export class Gdoc extends BaseEntity implements OwidGdocInterface {
             dodDocumentErrors.push(...errors)
         }
 
+        const faqs = this.content.faqs
```

Comment:
Hmmm, the dod document doesn't do this either, but we're not tracking links in these articles. oops!

We should fix that because we're counting on the links table being reliable for our content graph / 404 tracker.

Reply:
You mean that they should be included in the link tracking a few lines above? Can you outline what needs to be done? Then I could add it for both

Reply:
I think adding them to `blocksToTraverse` in `Gdoc.tsx` ought to be all you need to do.